### PR TITLE
CB-5547 added notification for user on whether repair can start or not

### PR DESCRIPTION
**Pull request by lnardai**
- added new graceful state when we can't start the repair
- based on sodre's branch added notification so user is aware that no repair was started

### CODE REVIEW

File: core/src/main/java/com/sequenceiq/cloudbreak/service/cluster/ClusterRepairService.java [hunk 1]
```diff
@@ -362,12 +364,14 @@ private void triggerRepairOrThrowBadRequest(Long stackId, Result<Map<HostGroupNa
             boolean removeOnly,
             Set<String> recoveryMessageArgument) {
         if (repairValidationResult.isError()) {
+            eventService.fireCloudbreakEvent(stackId, RECOVERY, CLUSTER_MANUALRECOVERY_COULD_NOT_START, repairValidationResult.getError().getValidationErrors());
```
Comment (foldik) [inline core/src/main/java/com/sequenceiq/cloudbreak/service/cluster/ClusterRepairService.java]: Can you extend the unit test as well?

Comment (lnardai) [reply core/src/main/java/com/sequenceiq/cloudbreak/service/cluster/ClusterRepairService.java]: On hold because Unit test are broken.

Comment (lnardai) [reply core/src/main/java/com/sequenceiq/cloudbreak/service/cluster/ClusterRepairService.java]: Added unit tests.

File: core/src/test/java/com/sequenceiq/cloudbreak/service/cluster/ClusterRepairServiceTest.java [hunk 2]
```diff
@@ -237,6 +241,7 @@ public void testCannotRepairGatewayWithoutExternalDatabase() throws CloudbreakIm
         boolean result = underTest.canRepairAll(stack);
 
         assertFalse(result);
+        //eventService.fireCloudbreakEvent(stackId, RECOVERY, CLUSTER_MANUALRECOVERY_COULD_NOT_START, repairValidationResult.getError().getValidationErrors());
```
Comment (foldik) [inline core/src/test/java/com/sequenceiq/cloudbreak/service/cluster/ClusterRepairServiceTest.java]: Do we need that in the future?

**Pull request merged**